lms/djangoapps/certificates/migrations/0020_certificatehtmlviewconfiguration_data.py

```diff
@@ -16,12 +16,12 @@ def forwards(self, orm):
             "default": {
                 "accomplishment_class_append": "accomplishment-certificate",
                 "platform_name": "edX",
```

Comment:
You should change the `platform_name` value, as well -- "edX" is a trademark of our company, and we shouldn't encourage others to use it.